common/shared.c

```diff
@@ -357,7 +365,7 @@ int ft_init_ep(void)
 int ft_init_av(void)
 {
 	size_t addrlen;
-	int ret;
+	ssize_t ret;
```

Comment:
Github won't let me comment on lines that weren't changed, but changing this from int->ssize_t also requires changing the format specifier in the printf calls below (lines 376 and 404) from `%d` to `%zd`. Found due to compiler warning.